examples/example-progs/moses-ann-pole1.cc

```diff
@@ -22,14 +22,14 @@
 #include <iostream>
 #include <opencog/util/mt19937ar.h>
 #include <opencog/util/Logger.h>
-#include <opencog/combo/interpreter/eval.h>
-
-#include <opencog/moses/deme/deme_expander.h>
-#include <opencog/moses/metapopulation/metapopulation.h>
-#include <opencog/moses/moses/moses_main.h>
-#include <opencog/moses/representation/representation.h>
-#include <opencog/moses/optimization/optimization.h>
-#include <opencog/moses/scoring/scoring_base.h>
+#include <opencog/asmoses/combo/interpreter/eval.h>
+
+#include <opencog/asmoses/moses/deme/deme_expander.h>
+#include <opencog/asmoses/moses/metapopulation/metapopulation.h>
+#include <opencog/asmoses/moses/moses/moses_main.h>
+#include <opencog/asmoses/moses/representation/representation.h>
+#include <opencog/asmoses/moses/optimization/optimization.h>
+#include <opencog/asmoses/moses/scoring/scoring_base.h>
 
```

Comment:
This doesn't look right. I think you want to have
```
#include <opencog/asmoses/representation/representation.h>
```
instead of
```
#include <opencog/asmoses/moses/representation/representation.h>
```

Reply:
I just added asmoses as a top-level directory without changing the previous directory structure. Also moses inside asmoses directory is one of the different libraries (others being combo, reduct..etc). Hence, I think having every moses related code in one directory (moses) makes sense.

Reply:
> I just added asmoses as a top-level directory without changing the previous directory structure. Also moses inside asmoses directory is one of the different libraries (others being combo, reduct..etc). Hence, I think having every moses related code in one directory (moses) makes sense.

OK, that sounds reasonable. I was surprised/confused by how many `#include` changes were needed.